### PR TITLE
Un-indent C++ namespaces. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: tests/sdl2_net_client.c [hunk 1]
```diff
@@ -46,11 +46,9 @@ void finish(int result) {
...
   }
 #ifdef __EMSCRIPTEN__
-  REPORT_RESULT(result);
-  emscripten_force_exit(result);
-#else
-  exit(result);
+  emscripten_cancel_main_loop();
 #endif
+  exit(result);
 }
```
Comment (kripken) [inline tests/sdl2_net_client.c:52]: unrelated change? from here the patch seems to be general cleanups and not C++ namespaces as the title says

Comment (sbc100) [reply tests/sdl2_net_client.c]: Ooops, sorry about that.  rebased

**Review by kripken — APPROVED** [review]
